lib/susedistribution.pm

```diff
@@ -613,6 +613,27 @@ sub init_consoles {
                     username => 'root'
                 });
         }
+        elsif (check_var('BACKEND', 'pvm_hmc')) {
+        $self->add_console('vios-ssh', 'ssh-xterm', {
+            hostname   => get_required_var('_SECRET_VIOS_HOSTNAME'),
+            password   => get_required_var('_SECRET_VIOS_PASSWORD'),
+            username   => get_var('_SECRET_VIOS_USERNAME', 'padmin'),
+            persistent => 1});
+        $self->add_console('root-ssh', 'ssh-xterm', {
+            hostname => get_required_var('SUT_IP'),
+            password => get_required_var('ROOT_PASSWORD'),
+            user     => 'root',
+            serial   => 'mknod -m 0666 /dev/sshserial p; while true; do cat /dev/sshserial; done'});
+        $self->add_console(
+                'installation',
+                'vnc-base',
+                {
+                    hostname => $hostname,
+                    port => 1,
+                    password => get_required_var('ROOT_PASSWORD'),
+                    #password => $testapi::password
```

Comment:
In this case, we're talking about the root password of the SUT right? if so
```suggestion
                    password => $testapi::password,
```

otherwise, for this case, you will need to document the variables too, and give them a better name